hoot-core/src/main/cpp/hoot/core/Hoot.cpp

```diff
@@ -129,7 +128,7 @@ void Hoot::_init()
   Log::getInstance().setLevel(Log::Info);
 }
 
-void Hoot::loadLibrary(QString name)
+void Hoot::loadLibrary(const QString& name)
```

Comment:
Just a bad habit of fixing parameters to constant reference instead of pass by value when not needed.